### ISSUE TITLE
Screen reader accessibility

### ISSUE THREAD

**Issue by theiliad** [carbon-design-system/carbon-charts#65]
Charts should be functional when being viewed by screen readers.

***JAWS* would be a good tool to test with, or the *Mac VoiceOver* tool:**
https://www.freedomscientific.com/Products/Blindness/JAWS

**Here's a good read about SVG accessibility:**
https://css-tricks.com/accessible-svgs/

**ARIA Screen Reader Implementors Guide:**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Screen_Reader_Implementors_Guide

**ARIA Label**:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

**ARIA Live Regions**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**ARIA Region:**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role

**ARIA Labelled By:**
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

**Tabindex:**
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

**Comment by theiliad** [comment]
A fair amount of this work has been finished, we should open issues for specific items if anything comes up
